torch/_inductor/triton_heuristics.py

```diff
@@ -362,16 +363,17 @@ def kernel_call():
                     {**dict(zip(self.arg_names, args)), **launcher.config.kwargs}
```

Comment:
Hmm, I'm not really sure what this pre_hook stuff is. I assume I need to put `**kwargs` here too, how to trigger/test this?